packages/text-editor/src/components/extension/nodeHighlight.ts

```diff
@@ -237,7 +227,7 @@ const createDecorations = (
           range.to,
           mergeAttributes(
             attributes,
-            nodeUuid === storage.activeNodeUuid ? { class: 'text-editor-highlighted-node-selected' } : {}
+            options.isActiveNode?.(nodeUuid) === true ? { class: 'text-editor-highlighted-node-selected' } : {}
```

Comment:
Why this code is not in `generateAttributes`?

Reply:
Updated